service-worker.js

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = 1;
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
+
+var preCachedRessources = [
+  {% for page in site.pages %}
+  '{{ page.url }}',
+  {% endfor %}
+];
+
+toolbox.precache(preCachedRessources);
+
+toolbox.router.get('/*', toolbox.cacheFirst);
+toolbox.router.get('/*', toolbox.cacheFirst, { origin: "cdnjs.cloudflare.com" });
```

Comment:
You'd probably want to configure a different cache name here the runtime CDN caches, so that they're reused indefinitely and don't become irrelevant whenever you change the default `toolbox.cache.name` value above.

If you're using `cacheFirst` strategy for them, please make sure that all of the URLs that they refer to are versioned. E.g. https://unpkg.com/react@15.3.1/dist/react.min.js includes `15.3.1` in the URL, so it's fine to use `cacheFirst`. But if you request URLs like https://unpkg.com/react/dist/react.min.js, once it's cached for the first time, the initial version will be reused indefinitely (at least until the cache name changes).